src/components/addDesc.js

```diff
@@ -1,11 +0,0 @@
-import axios from 'axios';
-
-const addDesc = (desc, id) => {
-    return axios.put('http://localhost:3032/api/animals/'+desc+'&'+id).then(res => {
-        return res.data
-    }).catch(error => {
-        console.log(error)
-    })
-}
-
-export default addDesc
```

Comment:
this isn't technically a component, but rather a helper function. I like that you broke it out, but I would put it in a utils folder instead of the components folder